docs/develop/configure-your-add-in-to-use-a-shared-runtime.md

```diff
@@ -0,0 +1,221 @@
+       ...
+   ```
+
+4. If you generated an Excel add-in with custom functions, find the `<Page>` element. Then change the source location from **Functions.Page.Url** to **Taskpane.Url**.
+
+   ```xml
+   <AllFormFactors>
+   ...
+   <Page>
+     <SourceLocation resid="Taskpane.Url"/>
+   </Page>
+   ...
+   ```
+
+5. Find the `<FunctionFile ...>` tag and change the `resid` from **Commands.Url** to  **Taskpane.Url**. Note that if you don't have action commands, you won't have a **FunctionFile** entry, and can skip this step.
+
+    ```xml
+    </GetStarted>
+    ...
+    <FunctionFile resid="Taskpane.Url"/>
+    ...
+    ```
+
+6. Save the **manifest.xml** file.
+
+## Configure the webpack.config.js file
+
+The **webpack.config.js** will build multiple runtime loaders. You need to modify it to load only the shared JavaScript runtime via the **taskpane.html** file.
+
+1. Start Visual Studio Code and open the Excel or PowerPoint add-in project you generated.
+2. Open the **webpack.config.js** file.
+3. If your **webpack.config.js** file has the following **functions.html** plugin code, remove it.
+
+    ```javascript
+    new HtmlWebpackPlugin({
+        filename: "functions.html",
+        template: "./src/functions/functions.html",
+        chunks: ["polyfill", "functions"]
+      })
+    ```
+
+4. If your **webpack.config.js** file has the following **commands.html** plugin code, remove it.
+
+    ```javascript
+    new HtmlWebpackPlugin({
+        filename: "commands.html",
+        template: "./src/commands/commands.html",
+        chunks: ["polyfill", "commands"]
+      })
+    ```
+
+5. If your project used either the **functions** or **commands** chunks, add them to the chunks list as shown next (the following code is for if your project used both chunks).
+
+    ```javascript
+      new HtmlWebpackPlugin({
+        filename: "taskpane.html",
+        template: "./src/taskpane/taskpane.html",
+        chunks: ["polyfill", "taskpane", "commands", "functions"]
+      })
+    ```
+
+6. Save your changes and rebuild the project.
+
+   ```command line
+   npm run build
+   ```
+
+> [!NOTE]
+> If your project has a **functions.html** file or **commands.html** file, they can be removed. The **taskpane.html** will load the **functions.js** and **commands.js** code into the shared JavaScript runtime via the webpack updates you just made.
+
+## Test your Office Add-in changes
+
+You can confirm that you are using the shared JavaScript runtime correctly by using the following instructions.
+
+1. Open the **manifest.xml** file.
+2. Find the `<Control xsi:type="Button" id="TaskpaneButton">` section and change the following `<Action ...>` XML.
+    
+    from:
+    
+    ```xml
+    <Action xsi:type="ShowTaskpane">
+      <TaskpaneId>ButtonId1</TaskpaneId>
+      <SourceLocation resid="Taskpane.Url"/>
+    </Action>
+    ```
+    
+    to:
+    
+    ```xml
+    <Action xsi:type="ExecuteFunction">
+    <FunctionName>action</FunctionName>
```

Comment:
Suggest indenting the <FunctionName> line, between the <Action></Action> elements: 
```suggestion
      <FunctionName>action</FunctionName>
```